customers/customers.go

```diff
@@ -12,17 +14,93 @@ type (
 		Name        string                 `json:"name,omitempty"`
 		Phone       *common.Phone          `json:"phone,omitempty"`
 		Metadata    map[string]interface{} `json:"metadata,omitempty"`
+		Default     string                 `json:"default,omitempty"`
 		Instruments []string               `json:"instruments,omitempty"`
 	}
 
 	GetCustomerResponse struct {
 		HttpMetadata common.HttpMetadata
-		Id           string                     `json:"id,omitempty"`
-		DefaultId    string                     `json:"nas,omitempty"`
-		Email        string                     `json:"email,omitempty"`
-		Name         string                     `json:"name,omitempty"`
-		Phone        *common.Phone              `json:"phone,omitempty"`
-		Metadata     map[string]interface{}     `json:"metadata,omitempty"`
-		Instruments  []common.InstrumentDetails `json:"instruments,omitempty"`
+		Id           string                       `json:"id,omitempty"`
+		Email        string                       `json:"email,omitempty"`
+		Default      string                       `json:"default,omitempty"`
+		Name         string                       `json:"name,omitempty"`
+		Phone        *common.Phone                `json:"phone,omitempty"`
+		Metadata     map[string]interface{}       `json:"metadata,omitempty"`
+		Instruments  []CustomerInstrumentResponse `json:"instruments,omitempty"`
```

Comment:
This is already defined on Instruments, so why just not import?
Also add some tests to confirm that this works

Reply:
👍🏽